docs/language/values-and-types.mdx

```diff
@@ -1352,6 +1352,28 @@ booleans[0] = true
 
   // `values` has type [Int] and is `[42, 23]`
   ```
+  
+- `cadence•fun contains(key: K): Bool`
```

Comment:
I didn't have much time to think about naming when I opened the issue, but I thought about it a bit more and had a look around: It might be a good idea to rename the function `containsKey` , to be more explicit what is being checked ("contains" could also be read as checking if the dictionary contains the given value as a value

`containsKey` is also the name used in Java, Kotlin, C#, Groovy, etc.

Could you please update the documentation here and the implementation below? 
Sorry for the inconvenience!

Reply:
agreed, done!